ci/ci/github.py

```diff
@@ -51,6 +51,21 @@ class GithubStatus(Enum):
     FAILURE = 'failure'
 
 
+def github_status(state: str) -> GithubStatus:
+    """
+    Converts a state for a commit status (https://docs.github.com/en/graphql/reference/enums#statusstate)
+    or a conclusion for a check (https://docs.github.com/en/graphql/reference/enums#checkconclusionstate)
+    from the GraphQL API to a GithubStatus.
+    """
+    if state in {"PENDING", "EXPECTED", "ACTION_REQUIRED", "STALE"}:
+        return GithubStatus.PENDING
+    elif state in {"FAILURE", "ERROR", "TIMED_OUT", "CANCELLED", "STARTUP_FAILURE", "SKIPPED"}:
+        return GithubStatus.FAILURE
+    elif state in {"SUCCESS", "NEUTRAL"}:
+        return GithubStatus.SUCCESS
```

Comment:
You need this for the lints to pass.
```suggestion
    if state in {"FAILURE", "ERROR", "TIMED_OUT", "CANCELLED", "STARTUP_FAILURE", "SKIPPED"}:
        return GithubStatus.FAILURE
    if state in {"SUCCESS", "NEUTRAL"}:
        return GithubStatus.SUCCESS
```